indicators/McAfee-Virus-8wer3hwa.yml

```diff
@@ -0,0 +1,33 @@
+title: McAfee Virus Phishing Kit 8wer3hwa
+description: Detects phishing pages mimicking McAfee with suspicious behavior.

```

Comment:
Needs more of a description for example,
Detects phishing pages mimicking McAfee detecting a virus on your computer using a suspicious domain checker and a keywords in the domain such as "virus detected or mcafee support. "

Reply:
I will add more of description!